client/java/src/main/java/org/jolokia/client/request/ValidatingResponseExtractor.java

```diff
@@ -54,10 +54,14 @@ public ValidatingResponseExtractor(int... pCodesAllowed) {
     public <RESP extends J4pResponse<REQ>, REQ extends J4pRequest> RESP extract(REQ pRequest,
                                                                       JSONObject pJsonResp)
             throws J4pRemoteException {
-
-        int status = pJsonResp.containsKey("status") ?
-                ((Long) pJsonResp.get("status")).intValue() :
-                0;
+	
+	int status = 0;
+	if( pJsonResp.containsKey("status")) {
+		Object o = pJsonResp.get("status");
+		if( o instanceof Long ) {
+			status = ((Long)o).intValue();
+		}
```

Comment:
The question is still what we should when status in **not** a long. I think its better to throw a `J4pRemoteException` at this case (including the value and type of the status) instead of silently ignoring it.

@robstryker wdyt ?